lib/misty/http/net_http.rb

```diff
@@ -6,12 +6,12 @@ def self.http_request(uri, options = {})
         http_options = {}
         if uri.scheme == 'https'
           http_options[:use_ssl] = true
-          if options[:ssl_verify_mode] && options[:ssl_verify_mode] == false
+          if options.fetch(:ssl_verify_mode, true) == false
             http_options[:verify_mode] = OpenSSL::SSL::VERIFY_NONE
           end
         end
 
-        Net::HTTP.start(uri.host, uri.port, http_options) do |connection|
+        Net::HTTP.start(uri.host, uri.port, :ENV, http_options) do |connection|
```

Comment:
According to the reference [1], `:ENV` is the default anyway and in which case that's not needed

[1] https://ruby-doc.org/stdlib-2.4.1/libdoc/net/http/rdoc/Net/HTTP.html#method-c-new